riptide-opentelemetry/src/main/java/org/zalando/riptide/opentelemetry/OpenTelemetryPlugin.java

```diff
@@ -0,0 +1,107 @@
+package org.zalando.riptide.opentelemetry;
+
+import com.google.common.collect.Multimaps;
+import io.opentelemetry.api.GlobalOpenTelemetry;
+import io.opentelemetry.api.trace.Span;
+import io.opentelemetry.api.trace.SpanKind;
+import io.opentelemetry.api.trace.Tracer;
+import io.opentelemetry.context.Context;
+import io.opentelemetry.context.Scope;
+import io.opentelemetry.context.propagation.TextMapPropagator;
+import org.springframework.http.client.ClientHttpResponse;
+import org.zalando.fauxpas.ThrowingBiConsumer;
+import org.zalando.riptide.Attribute;
+import org.zalando.riptide.Plugin;
+import org.zalando.riptide.RequestArguments;
+import org.zalando.riptide.RequestExecution;
+import org.zalando.riptide.opentelemetry.span.CompositeSpanDecorator;
+import org.zalando.riptide.opentelemetry.span.ErrorSpanDecorator;
+import org.zalando.riptide.opentelemetry.span.HttpHostSpanDecorator;
+import org.zalando.riptide.opentelemetry.span.HttpMethodSpanDecorator;
+import org.zalando.riptide.opentelemetry.span.HttpPathSpanDecorator;
+import org.zalando.riptide.opentelemetry.span.HttpStatusCodeSpanDecorator;
+import org.zalando.riptide.opentelemetry.span.PeerHostSpanDecorator;
+import org.zalando.riptide.opentelemetry.span.SpanDecorator;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+import java.util.function.BiConsumer;
+import javax.annotation.Nonnull;
+
+public class OpenTelemetryPlugin implements Plugin {
```

Comment:
Right now we can only extend the set of span decorators, but we cannot override the default set like we can with the OpenTracing plugin's [`withSpanDecorators`](https://github.com/zalando/riptide/blob/main/riptide-opentracing/src/main/java/org/zalando/riptide/opentracing/OpenTracingPlugin.java#L150).

Reply:
Added